devicon.json

```
@@ -8403,16 +8427,16 @@
             "frontend"
         ],
         "versions": {
-          "svg": [
-              "original",
-              "original-wordmark",
-              "plain",
-              "plain-wordmark"
-          ],
-          "font": [
-              "plain",
-              "plain-wordmark"
-          ]
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
```

Right! Let's try this.
```suggestion
          "svg": [
              "original",
              "original-wordmark",
              "plain",
              "plain-wordmark"
          ],
          "font": [
              "plain",
              "plain-wordmark"
          ]
```